cypress/e2e/date-info.cy.ts

```diff
@@ -0,0 +1,46 @@
+import { calculateMondayOffset } from "lib/integration/common";
+import { DateTime } from "luxon";
+
+describe("Date Info", () => {
+    const today = DateTime.now().startOf("day");
```

Comment:
I think we need to set the timezone here (and possibly other places in the codebase) to avoid some pitfalls

Reply:
Yeah, feel like this is a bit out of scope here, but I want to setup a DateTime provider that sets up time zone for the entire application